uniffi_bindgen/src/interface/mod.rs

```diff
@@ -119,6 +119,8 @@ impl<'ci> ComponentInterface {
             println!("{}", remaining);
             bail!("parse error");
         }
+        // Unconditionally add the String type, which is used by the error handling
```

Comment:
Specifically it's used by *panic* handling, so we need it even if the interface doesn't define any user-level errors.